tests/App/config/config.yml

```diff
@@ -0,0 +1,34 @@
+framework:
+    assets:
+        enabled: true
+    form:
+        enabled: true
+    session:
+        handler_id: session.handler.native_file
+        storage_id: session.storage.mock_file
+        name: MOCKSESSID
+    test: true
+    translator:
+        enabled: true
+
+twig:
+    exception_controller: null
+    strict_variables: false
+
+sonata_admin:
+    options:
+        legacy_twig_text_extension: false
```

Comment:
This produces deprecation warnings in the output of the test suite:
`The child node "legacy_twig_text_extension" at path "sonata_admin.options" is deprecated since sonata-project/admin-bundle 3.70 and will be removed in 4.0`. I think that's an issue with the admin bundle added in https://github.com/sonata-project/SonataAdminBundle/pull/6145 , cc @wbloszyk . The issue is that deprecation seems unfixable.

Reply:
Yes, I apparently it's unavoidable to get a deprecation warning, if you don't set it, you'll get:
```Using `true` as value for "sonata_admin.options.legacy_twig_text_extension" option is deprecated since sonata-project/admin-bundle 3.64. You should set it to `false`, which will be the default value since version 4.0.```.

Reply:
This option by default is `true`. In next major it will be deleted (work as always false). IMO `trigger_error` in validate if true should be set.